### DISCUSSION TITLE
Can I change the root label in global navigation sidebar?

### DISCUSSION THREAD

**Discussion by srob411** [requarks/wiki#3200] [Help / Questions] [answered]
Hi all!

My first post here... Managed to get wiki.js running in a Docker container on Debian 10. Now the amount of content is growing fast. Some day I saw in the global navigation sidebar this:

![image](https://user-images.githubusercontent.com/80406571/110646035-ca1f1e00-81b6-11eb-8f6c-12d8afb633d4.png)

Where does this "/ (root)" come from? I have no idea how I managed to create it. And I can't find any way to change it. Is there a way to do so?

TIA Robert

--

OS: Debian 10 and Docker
Wiki.js version: 2.5.170
Database engine: PostgreSQL 12.5

**Comment by NGPixel** [comment, marked as answer]
It's not something that can be changed dynamically. It's part of the locale strings.

**Reply by srob411** [reply]
Oh, that's a pity! Everybody is asking me if that is an error (they are expecting something like slash only, "Start" or "Home").

Thank you for the hint!

**Reply by adenoz** [reply]
I know this is already answered but I also started my wiki recently and thought to suggest you might want to choose to keep the 'Main Menu' item toggled rather than 'Browse'. I found that after customising it in the 'Navigation' section in the admin area you can get a really nice sidebar going and you can name the items anything you want, what link to use and choose icons and separators etc.

You might already know the above but I thought to mention it as it took me a little while to work that part out. Dramatically increased the quality of the user experience for me.

**Reply by srob411** [reply]
Not sure what is meant by 'keeping the main menu item toggled'... Is it that I can toggle between the treeview and a custom navigation in the sidebar? That's what I have setup. The custom navigation will be developed over time, depending on the needs of the colleagues in our NGO. But the treeview will offer access to the structure of the complete wiki tree, which might also be important for certain wiki users.

In the meantime, just this morning--without having any clue if it has been there before--I saw this for the first time:

![wiki-root2](https://user-images.githubusercontent.com/80406571/112320168-0cfced80-8caf-11eb-90aa-86d9ce4fa5b5.png)

Even worse than 'root'. If there really is no means to change that, it hopefully will disappear in the upcoming V3.0...

**Reply by felagund** [reply]
> It's not something that can be changed dynamically. It's part of the locale strings.

So in principle that should be translatable? I just tried for Czech, which claims to be 100 % translated and when I look for "sidebar", there is nothing to translate
. However, there is (root) that is translated. Is there not a bug that caused this string to be left untranslated?

**Reply by NGPixel** [reply]
Not all strings are translatable. This is being addressed in the upcoming 3.0 release.
